.travis.yml

```diff
@@ -0,0 +1,11 @@
+sudo: false
+language: python
+python:
+  - 3.3
+  - 3.4
+  - 3.5
+  - pypy3
+install:
+  - pip install flake8
+script:
+  - flake8 --ignore E501 homu
```

Comment:
You could also make it such that the longest line currently is what you set for max-line-length.